src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectDefaults.cs

```diff
@@ -28,6 +28,11 @@ public static class OpenIdConnectDefaults
         /// </summary>
         public static readonly string CookieNoncePrefix = ".AspNetCore.OpenIdConnect.Nonce.";
 
+        /// <summary>
+        /// The property for the prompt value being used when redirecting to the authentication provider.
+        /// </summary>
+        public static readonly string PromptPropertiesKey = "OpenIdConnect.Prompt";
```

Comment:
I'd rather use the existing constants like OpenIdConnectParameterNames.Prompt.

Reply:
Hmm, I wasn’t even aware that such a type existed! This would mean that people would need a using for `Microsoft.IdentityModel.Protocols.OpenIdConnect` though which—at least to me—gives off an internal feeling, and also makes discoverability rather difficult.

And I’m not sure if that would be safe against conflicts since the property key would then be a literal `"prompt"`. I understood the `"OpenIdConnect.~"` pattern as a way to make sure that these scheme-specific configs wouldn’t conflict with other items.